tests/clocks_rects_picosvg.ttx

```diff
@@ -64,7 +64,7 @@
+      <![CDATA[<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1"><defs><linearGradient id="g1" x1="64" y1="19.751" x2="64" y2="111.99" gradientUnits="userSpaceOnUse"><stop offset="0.321" stop-color="#ECEFF1"/><stop offset="1" stop-color="#B0BEC5"/></linearGradient></defs><g id="glyph2" transform="matrix(0.781 0 0 0.781 0 -100)"><path d="M124,64 A60 60 0 1 1 4,64 A60 60 0 1 1 124,64 Z" fill="url(#g1)" id="glyph2::0"/><path d="M63.8,63.98 H64.2 C66.3,63.98 68,62.28 68,60.18 V27.78 C68,25.68 66.3,23.98 64.2,23.98 H63.8 C61.7,23.98 60,25.68 60,27.78 V60.18 C60,62.28 61.7,63.98 63.8,63.98 Z" fill="blue"/><path d="M63.7,63.88 L64.04,64.09 C65.83,65.19 68.17,64.64 69.27,62.85 L80.61,44.48 C81.71,42.69 81.16,40.35 79.37,39.25 L79.03,39.04 C77.24,37.94 74.9,38.49 73.8,40.28 L62.47,58.65 C61.37,60.44 61.92,62.78 63.7,63.88 Z" fill="green"/><path d="M71,64 A7 7 0 1 1 57,64 A7 7 0 1 1 71,64 Z" fill="#424242" id="glyph2::3"/><path d="M67,64 A3 3 0 1 1 61,64 A3 3 0 1 1 67,64 Z" fill="white" id="glyph2::4"/><path d="M19.89,64.13 A4 4 0 1 1 11.89,64.13 A4 4 0 1 1 19.89,64.13 Z" fill="#757575" id="glyph2::5"/><use xlink:href="#glyph2::5" x="48" y="-48"/><use xlink:href="#glyph2::5" x="48" y="48"/><use xlink:href="#glyph2::5" x="96"/><path d="M64,8 C94.88,8 120,33.12 120,64 S94.88,120 64,120 S8,94.88 8,64 S33.12,8 64,8 M64,4 C30.86,4 4,30.86 4,64 S30.86,124 64,124 S124,97.14 124,64 S97.14,4 64,4 L64,4 Z" fill="#424242" opacity="0.2" id="glyph2::9"/></g><g id="glyph3" transform="matrix(0.781 0 0 0.781 0 -100)"><use xlink:href="#glyph2::0"/><path d="M63.8,63.98 H64.2 C66.3,63.98 68,62.28 68,60.18 V27.78 C68,25.68 66.3,23.98 64.2,23.98 H63.8 C61.7,23.98 60,25.68 60,27.78 V60.18 C60,62.28 61.7,63.98 63.8,63.98 Z" fill="purple"/><path d="M63.78,63.81 L63.97,64.16 C64.98,66 67.29,66.68 69.13,65.67 L88.07,55.31 C89.91,54.3 90.59,51.99 89.58,50.15 L89.39,49.8 C88.38,47.96 86.07,47.28 84.23,48.29 L65.29,58.65 C63.45,59.66 62.77,61.97 63.78,63.81 Z" fill="red"/><use xlink:href="#glyph2::3"/><use xlink:href="#glyph2::4"/><use xlink:href="#glyph2::5"/><use xlink:href="#glyph2::9"/></g></svg>]]>
```

Comment:
It is best to add a more concise test case, which is a good choice:

```svg
<svg width="200" height="200" viewBox="0 0 200 200" version="1.1" xmlns="http://www.w3.org/2000/svg">
    <circle fill="#F49924" cx="100" cy="100" r="100" />
    <ellipse fill="#7E4418" cx="59.545" cy="71.818" rx="15" ry="21.818" />
    <ellipse fill="#7E4418" cx="140.455" cy="71.818" rx="15" ry="21.818" />
</svg>
```

Reply:
ok, I'll add that.